src/Components/Todo.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+class Todo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      clicked: false
+    };
+  }
+
+  handleClick = () => {
+    const clicked = this.state.clicked;
+    this.setState({
+      clicked: !clicked
```

Comment:
I'm not 100% certain what is going on here but I think this is where you have a bug.